msteams-platform/tabs/how-to/using-teams-client-library.md

```diff
@@ -29,6 +29,40 @@ The remainder of this article walks you through the structure and latest updates
 
 TeamsJS v.2.0 introduces the ability for certain types of Teams apps to run across the Microsoft 365 ecosystem. The other Microsoft 365 application hosts (including Microsoft 365 app and Outlook) for Teams apps support a subset of the application types and capabilities you can build for the Teams platform. This support expands over time. For a summary of host support for Teams apps, see [TeamsJS capability support across Microsoft 365](../../m365-apps/teamsjs-support-m365.md).
 
+### Improve app performance with tree shaking
+
+Starting with TeamsJS version 2.31.0, the library is fully tree-shakeable. Tree shaking is a JavaScript optimization that eliminates unused code. By using tree shaking when an app is bundled for deployment you can reduce package size, which results in faster download and improved load time.
```

Comment:
How about (for better flow):

Starting with version 2.31.0, the TeamsJS library is fully tree-shakeable.

And also, always feel free to reference MDN webdocs from MS Learn docs to explain general concepts. (Microsoft has a formal content partnership w/MDN and has a seat on the MDN web docs product advisory board.)

 `[Tree shaking](https://developer.mozilla.org/docs/Glossary/Tree_shaking) is a JavaScript optimization that eliminates unused code.`

Reply:
Sounds good, updated in next commit.